### PR TITLE
Docs an example for triggers

### PR DESCRIPTION
**Pull request by chbatey**
The equivalent of event triggers in multi dc by sending a command back to self from event handler

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
I think we can merge this and I'll follow up with fixes given that chris is away this week. Let's hold off until I start working on it though.

File: akka-docs/src/main/paradox/typed/persistence-active-active-examples.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Scala
+:   @@snip [AuctionExample](/akka-persistence-typed-tests/src/test/scala/docs/akka/persistence/typed/AAAuctionExampleSpec.scala) { #commands }
+
+The events:
+
+Scala
+:   @@snip [AuctionExample](/akka-persistence-typed-tests/src/test/scala/docs/akka/persistence/typed/AAAuctionExampleSpec.scala) { #events }
+
+The winner does not have to pay the highest bud but only enough to beat the second highest so the `highestCounterOffer` is in the `AuctionFinished` event.
```
Comment (renatocaval) [inline akka-docs/src/main/paradox/typed/persistence-active-active-examples.md:26]:
```suggestion
The winner does not have to pay the highest bid but only enough to beat the second highest so the `highestCounterOffer` is in the `AuctionFinished` event. 
```

File: akka-docs/src/main/paradox/typed/persistence-active-active-examples.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+Scala
+:   @@snip [AuctionExample](/akka-persistence-typed-tests/src/test/scala/docs/akka/persistence/typed/AAAuctionExampleSpec.scala) { #setup }
+
+
+The auction moves through the following phases:
+
+Scala
+:   @@snip [AuctionExample](/akka-persistence-typed-tests/src/test/scala/docs/akka/persistence/typed/AAAuctionExampleSpec.scala) { #phase }
+
+The closing and closed states are to model waiting for all replicas to see the result of the auaction before
```
Comment (renatocaval) [inline akka-docs/src/main/paradox/typed/persistence-active-active-examples.md:52]:
```suggestion
The closing and closed states are to model waiting for all replicas to see the result of the auction before
```

File: akka-docs/src/main/paradox/typed/persistence-active-active-examples.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+The ordering between bids is crucial, therefore. We need to ensure that it is deterministic and does not depend on local state
+outside of our state class so that all replicas come to the same result. We define the ordering as this:
+
+ * A higher bid wins.
+ * If there's a tie between the two highest bids, the bid that was registered earlier wins. For that we keep track of the
+   (local) timestamp the bid was registered.
+ * We need to make sure that no timestamp is used twice in the same replica (missing in this example).
+ * If there's a tie between the timestamp, we define an arbitrary but deterministic ordering on the replicas, in our case
+   we just compare the name strings of the replicass. That's why we need to keep the identifier of the replica where a bid was registered
```
Comment (renatocaval) [inline akka-docs/src/main/paradox/typed/persistence-active-active-examples.md:82]:
```suggestion
   we just compare the name strings of the replicas. That's why we need to keep the identifier of the replica where a bid was registered
```

File: akka-docs/src/main/paradox/typed/persistence-active-active-examples.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+In the auction we want to ensure that all bids are seen before declaring a winner. That means that an auction can only be closed once
+all replicas have seen all bids.
+
+In the event handler above, when recovery is not running, it calls `eventTriggers`.
+
+Scala
+:   @@snip [AuctionExample](/akka-persistence-typed-tests/src/test/scala/docs/akka/persistence/typed/AAAuctionExampleSpec.scala) { #event-triggers }
+
+The event trigger uses the `ActiveActiveContext` to decide when to trigger the Finish of the action.
+When a replica saves the `AuctionFinished` event it checks whether it should close the auaction.
```
Comment (renatocaval) [inline akka-docs/src/main/paradox/typed/persistence-active-active-examples.md:101]:
```suggestion
When a replica saves the `AuctionFinished` event it checks whether it should close the auction.
```

**Review by octonato — APPROVED** [review]
LGTM, found a few typos though. Sent soem GitHub suggestions to fix them.

**Pull request merged**